src/components/Apis/ApisList/ApisListTable.tsx

```diff
@@ -0,0 +1,77 @@
+import {useEffect, useMemo, useState} from 'react';
+
+import {ApiItem} from '@models/api';
+import {ApisTableDataSourceItem} from '@models/dashboard';
+
+import {useAppSelector} from '@redux/hooks';
+
+import {ListTableColumnLabel} from '@components';
+
+import * as S from './ApisListTable.styled';
+import ApisListTableServicesTag from './ApisListTableServicesTag';
+
+interface IProps {
+  apis: ApiItem[];
+}
+
+const ApisListTable: React.FC<IProps> = props => {
+  const {apis} = props;
+
+  const selectedApi = useAppSelector(state => state.main.selectedApi);
+  const selectedApiKey = useMemo(() => {
```

Comment:
maybe this could be extracted in a `useSelectedApiKey` if it will be used in multiple places